data-base/db.go

```diff
@@ -39,42 +42,54 @@ func NewRecordDB(db *sqlx.DB) *RecordDB {
 }
 func (db *RecordDB) NewRecord(r *repo.Record) error {
 	err := db.Get(r, QueryCreate, r.ID, r.Type, r.CaesarShift, r.Result, r.CreatedAt, r.UpdatedAt)
+
 	if err != nil {
-		return err
+		return fmt.Errorf("error creating new record: %w", err)
 	}
+
 	return nil
 }
 
 func (db *RecordDB) GetRecord(id string) (repo.Record, error) {
-	var r repo.Record
-	err := db.Get(&r, QuerySingleRead, id)
+	var result repo.Record
+
+	err := db.Get(&result, QuerySingleRead, id)
 	if err != nil {
-		return repo.Record{}, err
+		return repo.Record{}, fmt.Errorf("error reading one record: %w", err)
 	}
-	return r, nil
+
+	return result, nil
 }
 
-func (db *RecordDB) GetRecords() ([]repo.Record, error) {
-	var r []repo.Record
-	err := db.Select(&r, QueryMultiRead)
+func (db *RecordDB) GetAllRecords() ([]repo.Record, error) {
+	var result []repo.Record
+	err := db.Select(&result, QueryMultiRead)
+
+	sort.Slice(result, func(i, j int) bool {
+		return result[i].CreatedAt > result[j].CreatedAt
+	})
+
 	if err != nil {
-		return []repo.Record{}, err
+		return result, fmt.Errorf("error reading all records: %w", err)
 	}
-	return r, err
+
+	return result, nil
 }
 
 func (db *RecordDB) UpdateRecord(r *repo.Record) error {
 	err := db.Get(r, QueryUpdate, r.Type, r.CaesarShift, r.Result, r.UpdatedAt, r.ID)
```

Comment:
It looks like passing `r.UpdatedAt` as `updated_at` will have no effect. Anyway, it's better to set it `updated_at = NOW()` in the SQL query or by trigger